assets/js/renderer.js

```diff
@@ -388,8 +388,31 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Instantiate the splitter.
+    const splitter = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: function() {
+            sessionStorage.setItem('splitSizes', JSON.stringify(splitter.getSizes()));
+        }
+    });
+
+    // Retrieve any saved resize settings from sessionStorage or else use our default setting.
+    function restoreSplitterSizes() {
+        let splitSizes;
```

Comment:
@johnmhoran One more thing I am noticing now, if you get an error in the try-catch then `splitSizes` is never set. One way around this is to assign the default value when you initialize `splitSizes`.

```
        let splitSizes = [18, 76];
        try {
            splitSizes = JSON.parse(sessionStorage.getItem('splitSizes')) || splitSizes;
        } catch (err) {
            console.log(err);
        }
        splitter.setSizes(splitSizes);
```